README.md

```diff
@@ -164,14 +182,38 @@ Or can be ignored by setting the `ignored_models` config
 
 > Note: With namespaces, wrap your model name in double-quotes (`"`): `php artisan ide-helper:models "API\User"`, or escape the slashes (`Api\\User`).
 
-For properly recognition of `Model` methods (i.e. `paginate`, `findOrFail`) you should extend `\Eloquent` or add
```

Comment:
This was deliberately removed:
- are we _really_ suggesting people should extend their models from `\Eloquent`? To the best of my knowledge, this is not necessary at all 🤔
- same with add `/** @mixin \Eloquent */`, this is nowadays always added?
  (note: the diff is hard to read, there two lines removed, the second one is much further down)